data/migrations/20200625220949_create_profile.js

```diff
@@ -23,5 +23,7 @@ exports.up = (knex) => {
 };
 
 exports.down = (knex) => {
-  return knex.schema.dropTableIfExists('profiles').dropTableIfExists('roles');
+  return knex.schema
+    .raw('DROP TABLE profiles CASCADE')
+    .raw('DROP TABLE roles CASCADE');
```

Comment:
Nice change to catch all relevant data on rollback.

Reply:
I learned something new here. Very cool change.